demo/simple.c

```diff
@@ -1,4 +1,8 @@
+
+#ifndef BUILD_OS_WIN
```

Comment:
@tysenmoore-xse @okbob 
Please don't use a check for the build system here and in other places but check for the header and function in configure.ac, either directly or "more portable" (reference and hints for later: https://www.mail-archive.com/groff@gnu.org/msg01309.html)
... but the direct check for the header and function in configure.ac and the check of the auto-generated define would still be much better than checking the build system (which should also allow to get rid of those "self-created" defines [and possible breaking of unknown build systems that worked before] via configure at all)

Reply:
Unfortunately I have not good enough experience with autoconf and with writing portable code. I wrote and maintain only this library. So I know it, so my knowledges are too low. If you have any more specific ideas about enhancing or cleaning related issues, please, send pull request.

Reply:
@GitMensch
I did not consider this approach only because:
- the header was always used in the source so I assumed that was expected
- the header is not available in Windows
- this was only used in the demo/simple files, not core files
I can have a look at a configure.ac change to support this.  Seems like this is would not have been a unique comment to my PR but rather the original code.